src/context/CapturesContext.js

```diff
@@ -237,27 +244,210 @@ export function CapturesProvider(props) {
     setCaptures([]);
     setPage(0);
     setCaptureCount(null);
+    setInvalidateCaptureCount(true);
+  };
+
+  /**  VERIFICATION METHODS **/
+
+  const resetSelection = () => {
+    setCapturesSelected([]);
+    setCaptureImageAnchor(undefined);
+  };
+
+  const clickCapture = (payload) => {
+    const { captureId, isShift, isCmd, isCtrl } = payload;
+    if (!isShift && !isCmd && !isCtrl) {
+      setCapturesSelected([captureId]);
+      setCaptureImageAnchor(captureId);
+    } else if (isShift) {
+      log.debug('press shift, and there is an anchor:', captureImageAnchor);
+      //if no anchor, then, select from beginning
+      let indexAnchor = 0;
+      if (captureImageAnchor !== undefined) {
+        indexAnchor = captures.reduce((a, c, i) => {
+          if (c !== undefined && c.id === captureImageAnchor) {
+            return i;
+          } else {
+            return a;
+          }
+        }, -1);
+      }
+      const indexCurrent = captures.reduce((a, c, i) => {
+        if (c !== undefined && c.id === captureId) {
+          return i;
+        } else {
+          return a;
+        }
+      }, -1);
+      const capturesSelected = captures
+        .slice(
+          Math.min(indexAnchor, indexCurrent),
+          Math.max(indexAnchor, indexCurrent) + 1
+        )
+        .map((capture) => capture.id);
+      setCapturesSelected(capturesSelected);
+    } else if (isCmd || isCtrl) {
+      // Toggle the selection state
+      let selectedImages;
+      if (capturesSelected.find((el) => el === captureId)) {
+        selectedImages = capturesSelected.filter(function (capture) {
+          return capture !== captureId;
```

Comment:
For consistency, since there are arrow functions being used in the previous line, we can stick to that. I think you can do something like:

```javascript
selectedImages = capturesSelected.filter((capture) => {
          return capture !== captureId;
   });
```